src/server/extensions/compatibility-layer/searx/bridge/runtime.py

```
@@ -0,0 +1,125 @@
+
+    def set(self, key=None, value=None, expire=None, **kwargs):
+        if key is None:
+            key = kwargs.get("name")
+        self._local[key] = value
+        stored = encode(value)
+        if stored is None:
+            return value
+        try:
+            rpc.call({"rpc": "cache", "op": "set", "key": str(key), "value": stored, "ttl": expire})
+        except Exception:
+            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)